commands/function.ts

```diff
@@ -0,0 +1,23 @@
+// @ts-nocheck
+const list = require('./function/list');
+const deploy = require('./function/deploy');
+const server = require('./function/server');
+const { i18n } = require('../lib/lang');
+
+const i18nKey = 'commands.function';
+
+exports.command = ['function', 'functions'];
+exports.describe = i18n(`${i18nKey}.describe`);
+
+exports.builder = yargs => {
+  yargs
+    .command({
+      ...list,
+      aliases: 'ls',
+    })
```

Comment:
Should we move this to the `command` field in the list subcommand?

Reply:
Ah yeah I saw you did that in another PR. I'll do the same 👍